components/features/inline-summary.tsx

```
@@ -11,12 +12,13 @@ import {
   SelectTrigger,
 } from "@/components/ui/select";
 import { Skeleton } from "@/components/ui/skeleton";
-import { UseQueryResult, useQuery } from "@tanstack/react-query";
+import { UseQueryResult } from "@tanstack/react-query";
 import useLocalStorage from "@/lib/hooks/use-local-storage";
 import { Response } from "../ai/response";
-import { useIsPremium } from "@/lib/hooks/use-is-premium";
-import { ChevronDown, ChevronUp, Globe, Database } from "lucide-react";
+import { ChevronDown, ChevronUp, Globe, Database, AlertCircle, Zap, Infinity } from "lucide-react";
```

Biome (2.1.2)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)